qingstor/sdk/client/remote_encryption_client.py

```diff
@@ -0,0 +1,32 @@
+import base64
+import hashlib
+
+from Crypto.Cipher import AES
+
+
+class RemoteEncryptionClient:
```

Comment:
Use `EncryptionClient` may be better ?

We should support all API list here: https://docs.qingcloud.com/qingstor/api/common/encryption.html#api